deploy-manage/security/_snippets/own-ca-warning.md

```diff
@@ -0,0 +1,5 @@
+:::{warning}
+Transport connections between {{es}} nodes are security-critical and you must protect them carefully. Malicious actors who can observe or interfere with node-to-node transport traffic can read or modify cluster data. A malicious actor who can establish a transport connection might be able to invoke system-internal APIs, including APIs that read or modify cluster data.
```

Comment:
I think we need to qualify that this means the _raw_ (i.e. plaintext, unencrypted) traffic. Without that, "node-to-node transport traffic" might also mean the encrypted traffic.